pkgs/by-name/ll/lldap/package.nix

```diff
@@ -11,6 +11,11 @@
 }:
 
 let
+  wasm-bindgen-95 = wasm-bindgen-cli.override {
+    version = "0.2.95";
+    hash = "sha256-prMIreQeAcbJ8/g3+pMp1Wp9H5u+xLqxRxL+34hICss=";
+    cargoHash = "sha256-6iMebkD7FQvixlmghGGIvpdGwFNLfnUcFke/Rg8nPK4=";
+  };
```

Comment:
is this an overlapping change from https://github.com/NixOS/nixpkgs/pull/374179 ?

Reply:
Sorry, I don't know how I managed to not pay attention to this.